plugins/wls.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+    name: wls
+spec:
+    version: v1.0.0
+    homepage: 'https://github.com/mohatb/kubectl-wls'
+    shortDescription: 'Windows-linux-shell allows you to start a shell on windows or linux nodes'
+    description: "kubectl-wls stands for kubectl Kubectl-Windows-Linux-Shell, it is an interactive shell script and also a kubectl plugin that allows you to access a kubernetes windows or linux nodes shell.\n"
+    platforms:
+        -
+            selector: {matchExpressions: [{key: os, operator: In, values: [darwin, linux]}]}
+            uri: 'https://github.com/mohatb/kubectl-wls/archive/v1.0.0.tar.gz'
```

Comment:
your source code says

> $(kubectl get node aksnpwin000000

have you actually tested this thoroughly?

Reply:
updated source with a variable instead

Reply:
yes, and I updated both yaml and the source